packages/flutter/lib/src/widgets/tap_and_drag_gestures.dart

```diff
@@ -1031,7 +1011,8 @@ class TapAndDragGestureRecognizer extends OneSequenceGestureRecognizer with _Tap
       // a sufficient global distance from the initial position to be considered a drag.
       // In this case since the gesture cannot be a tap, it defaults to a drag.
 
-      _pastSlopTolerance = _pastSlopTolerance || slopTolerance != null && _getGlobalDistance(event, _initialPosition) > slopTolerance!;
+      final double computedSlop = computeHitSlop(event.kind, gestureSettings);
```

Comment:
I have changed this slop.
Otherwise, `_pastSlopTolerance` will still false even if already accept drag(_hasSufficientGlobalDistanceToAccept) when use mouse drag.